params/config.go

```diff
@@ -819,7 +819,7 @@ func configTimestampEqual(x, y *uint64) bool {
 		return y == nil
 	}
 	if y == nil {
-		return x == nil
+		return false
```

Comment:
current code is more robust, new code depends on order. Neither is IMO more readable than the other. please revert

Reply:
done